### PR TITLE
chore: upgrade to prefect 1.4.1

### PR DESCRIPTION
**Pull request by Hamza442**

### CODE REVIEW

File: edx_prefectutils/mysql.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 """
 import os
 
-import mysql.connector
```
Comment (HassanJaveed84) [inline edx_prefectutils/mysql.py:6]: Curious about needing to change this ?

File: tox.ini [hunk 2]
```diff
@@ -22,4 +22,4 @@ commands =
     touch tests/__init__.py
     flake8 edx_prefectutils tests
     rm tests/__init__.py
-    isort --check-only --diff --recursive edx_prefectutils tests
```
Comment (HassanJaveed84) [inline tox.ini:25]: Do we not want to check the subdirectories(if any) ?

File: edx_prefectutils/mysql.py (hunk 1 above)
Comment (Hamza442) [reply edx_prefectutils/mysql.py:6]: @HassanJaveed84 so mysql.connector had a lot of dependency issues with prefect new version and it required pinning a lot of things so I thought rather doing that using a other mysql connector don't decency issue will be a better option.

File: tox.ini (hunk 2 above)
Comment (Hamza442) [reply tox.ini:25]: So in isort latest version we don't need to explicitly put `--recursive` option. It something that's now a default behaviour 
Details are mentioned here : https://pycqa.github.io/isort/docs/upgrade_guides/5.0.0.html
If we use this option we getting following warnings
`UserWarning: W0501: The following deprecated CLI flags were used and ignored: --recursive!`

**Pull request closed without merging**